### PR TITLE
Bug 1649400 - part 1: ship org.mozilla.fenix as a nightly product

### PR DESCRIPTION
**Pull request by JohanLorenzo**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: try_task_config.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (MihaiTabara) [inline try_task_config.json:1]: Nit: remove this before merge.

**Comment by JohanLorenzo** [comment]
I scheduled a nightly graph + a nightly-gp one and none of the decision tasks broke. I spot-checked the task configurations and it looks good. I cancelled both graphs to not ship anything until tomorrow, which is the scheduled date.

File: app/src/fenixProduction/res/values/static_strings.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <!-- Name of the application -->
+    <string name="app_name" translatable="false">Firefox Nightly</string>
```
Comment (liuche) [inline app/src/fenixProduction/res/values/static_strings.xml:7]: It looks like we'll be using the brand name "Firefox Nightly"!

I can't see the assets though, so I want to confirm that we're going to be shipping with the Firefox Nightly assets/branding (which is on fennecNightly flavor) and NOT the Firefox Preview assets/branding (which has been the fenixProduction flavor)

cc @pocmo @JohanLorenzo

Comment (pocmo) [reply app/src/fenixProduction/res/values/static_strings.xml:7]: @liuche It looks like we copied the fenixNightly assets over, but we may actually want the fennecNightly assets (outline logo vs. full logo)?

Comment (JohanLorenzo) [reply app/src/fenixProduction/res/values/static_strings.xml:7]: Discussed on Slack with @liuche, yesterday. Fixed in https://github.com/mozilla-mobile/fenix/pull/12173.

**Pull request merged**